ci/licenses.sh

```diff
@@ -122,7 +122,7 @@ function verify_licenses() (
 
   local actualLicenseCount
   actualLicenseCount="$(tail -n 1 flutter/ci/licenses_golden/licenses_flutter | tr -dc '0-9')"
-  local expectedLicenseCount=15 # When changing this number: Update the error message below as well describing all expected license types.
+  local expectedLicenseCount=12 # When changing this number: Update the error message below as well describing all expected license types.
```

Comment:
Is this simply a function of removing unused files?

Reply:
not sure what you mean by removing unused files. This make sure the license count does not change unexpectedly

Reply:
My assumption is that this is that changes made in thsi patch are reducing the number of licenses we're including, so we're lowering this number.

Is that accurate? We've simply deleted files with different copyright years or whatever, so we no longer have as many?

Reply:
that is correct